packages/source-iotsitewise/src/time-series-data/client/client.spec.ts

```diff
@@ -241,6 +243,7 @@ describe('getLatestPropertyDataPoint', () => {
       end: endDate,
       resolution,
       fetchMostRecentBeforeEnd: true,
+      aggregationType: AGGREGATE_TYPE,
```

Comment:
looks like we are still requesting aggregationType here even for 0 resolution

Reply:
great catch! updated :)

Reply:
fixed! thanks for catching that